types/standard-engine/index.d.ts

```diff
@@ -0,0 +1,153 @@
+// Type definitions for standard-engine 12.0
+// Project: https://github.com/flet/standard-engine
+// Definitions by: Piotr Błażejewicz (Peter Blazejewicz) <https://github.com/peterblazejewicz>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+import eslint = require('eslint');
+import { type } from 'os';
+
+export class Linter {
```

Comment:
The JS library exports this class as `linter` and not `Linter`, so I think you should not export this class directly because it implies that consumers of the library will be able to do write code like this:
```js
import standardEngine = require('standard-engine');
const l = new standardEngine.Linter();
```
which I think doesn't really work.

Reply:
@gabritto amended that part, thx for the review!